docs/cli/edgedb_toml.rst

```diff
@@ -0,0 +1,44 @@
+.. _ref_cli_edgedb_toml:
+
+===========
+edgedb.toml
+===========
+
+The ``edgedb.toml`` file is a key component of EdgeDB's project system. If this
+file is present in a directory, it signals to the CLI that the directory is an
```

Comment:
> it signals to the CLI

also to any client bindings ;)

Reply:
Oh, good point: I wonder if this should live outside of the CLI docs? Or at least also live somewhere about the general project structure of an EdgeDB project?

Reply:
I almost put it in the reference section instead, but I thought only the CLI cared about it. If client bindings also use it, I think it makes sense to put it there instead. Thanks, @fantix!